content/en/continuous_integration/setup_pipelines/customspans.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Custom spans
+kind: documentation
+further_reading:
+    - link: "/continuous_integration/setup_pipelines/customspans/"
+      tag: "Documentation"
+      text: "Troubleshooting CI"
+---
+
+Custom spans provide a way to trace individual commands in your CI pipelines, allowing you to measure the time your command takes without taking into account any setup or teardown actions that the job might have (for example, downloading Docker images or waiting for an available node in a Kubernetes-based infrastructure). These spans appear as part of the pipeline's trace:
+
+{{< img src="ci/ci-custom-spans.png" alt="Details for a single pipeline with custom spans" style="width:100%;">}}
+
+## Compatibility
+
+Custom spans work with the following CI providers:
+
+- Jenkins with Datadog plugin >= v3.2.0
+- CircleCI
```

Comment:
We should remove this or wait until we merge CircleCI documentation before merging this.